src/Components/Filtering/RangeFilter.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable no-unused-vars */
```

Comment:
delete these eslint disables too! If your code follows these guidelines it'll be cleaner.